src/prefect/tasks/aws/lambda_function.py

```diff
@@ -117,12 +117,19 @@ def run(self, credentials: dict = None):
                 with two keys: `ACCESS_KEY` and `SECRET_ACCESS_KEY` which will be
                 passed directly to `boto3`.  If not provided here or in context, `boto3`
                 will fall back on standard AWS rules for authentication.
+            - boto_kwargs (dict, optional): additional keyword arguments to pass to the
+                Task constructor that are forwarded as kwargs to the boto client
```

Comment:
Edit on the docstring, would be good to apply this everywhere (just commenting here).

```suggestion
            - boto_kwargs (dict, optional): additional keyword arguments to
                forward to the boto client.
```

Reply:
Just edited the boto_kwargs descriptions accordingly.